lib/allWarnings.js

```diff
@@ -0,0 +1,68 @@
+const summary = (text, obj) => {
+  if (text.length > 120) {
+    // eslint-disable-next-line no-param-reassign
+    obj.hasLongLine = obj.hasLongLine || text.length > 500;
```

Comment:
It's quite confusing to manipulate the object in here. Don't you think it's a better idea to put it into the class and set a `this.ranOnMinifiedFiles: boolean`

Reply:
Or you can keep the object if in future there will be stored more information

Reply:
Ok, I think I guess where you want to go, and I'm OK. This warning class could be used for way more tests. I'll change that.